index.html

```diff
@@ -319,47 +320,6 @@ <h2 class="title_contact">Contact me</h2>
 			</div>
 		</section>
 	</main>
```

Comment:
- [ ] [OPTIONAL] I would suggest you remove the form from the main tag and include inside a footer to make it more semantic.

Reply:
All are Done! Thank you! can I be approved, please?